test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -229,4 +229,156 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
+			}
+
+			// filter by resources on each numa zone
+			requiredResCnt1 := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("4"),
+				corev1.ResourceMemory: resource.MustParse("4Gi"),
+			}
+
+			requiredResCnt2 := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("5"),
+				corev1.ResourceMemory: resource.MustParse("5Gi"),
+			}
+
+			By("filtering available nodes with allocatable resources on at least one NUMA zone that can match request")
+			nrtCandidates = e2enrt.FilterNRTsEachRequestOnADifferentZone(nrtCandidates, requiredResCnt1, requiredResCnt2)
+			if len(nrtCandidates) < neededNodes {
+				Skip(fmt.Sprintf("not enough nodes with NUMA zones each of them can match requests: found %d, needed: %d", len(nrtCandidates), neededNodes))
+			}
+
+			// After filter get one of the candidate nodes left
+			nrtCandidateNames := e2enrt.AccumulateNames(nrtCandidates)
+			targetNodeName, ok := nrtCandidateNames.PopAny()
+			Expect(ok).To(BeTrue(), "cannot select a target node among %#v", nrtCandidateNames.List())
+			By(fmt.Sprintf("selecting node to schedule the pod: %q", targetNodeName))
+
+			By("Padding all other candidate nodes")
+			var paddingPods []*corev1.Pod
+			for _, nodeName := range nrtCandidateNames.List() {
+
+				nrtInfo, err := e2enrt.FindFromList(nrtCandidates, nodeName)
+				Expect(err).NotTo(HaveOccurred(), "missing NRT info for %q", nodeName)
+
+				baseload, err := nodes.GetLoad(fxt.K8sClient, nodeName)
+				Expect(err).ToNot(HaveOccurred(), "missing node load info for %q", nodeName)
+
+				baseloadResources := corev1.ResourceList{
+					corev1.ResourceCPU:    baseload.CPU,
+					corev1.ResourceMemory: baseload.Memory,
+				}
+
+				paddingResources, err := e2enrt.SaturateNodeUntilLeft(*nrtInfo, baseloadResources)
+				Expect(err).ToNot(HaveOccurred(), "could not get padding resources for node %q", nrtInfo.Name)
+
+				for idx, zone := range nrtInfo.Zones {
+					podName := fmt.Sprintf("padding%s-%d", nodeName, idx)
+					padPod := newPaddingPod(nodeName, zone.Name, fxt.Namespace.Name, paddingResources[zone.Name])
+
+					padPod, err = pinPodTo(padPod, nodeName, zone.Name)
+					Expect(err).NotTo(HaveOccurred(), "unable to pin pod %q to zone %q", podName, zone.Name)
+
+					err = fxt.Client.Create(context.TODO(), padPod)
+					Expect(err).NotTo(HaveOccurred(), "unable to create pod %q on zone %q", podName, zone.Name)
+
+					paddingPods = append(paddingPods, padPod)
+				}
+			}
+
+			By("Padding target node")
+			// Pad the zones so no one could handle both containers
+			paddingResources := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("6"),
+				corev1.ResourceMemory: resource.MustParse("6Gi"),
+			}
+			//calculate base load on the target node
+			targetNodeBaseLoad, err := nodes.GetLoad(fxt.K8sClient, targetNodeName)
+			Expect(err).ToNot(HaveOccurred(), "missing node load info for %q", targetNodeName)
+
+			targetNodeBaseLoad.Deduct(paddingResources)
```

Comment:
why deduct here?

Reply:
I thought I should take into consideration the resources needed for the node base load, but maybe I should use "deduct" just for one of the numa zones instead of on all of them.

Reply:
Wait, I didn't mean this is wrong. I just feel this is a bit hard to understand, so if you can clarify a bit the flow this can be enough to move forward (or at least to have a better review next cycle :) )

Reply:
But it is wrong ... 
At this point we know `targetNode` have at least two zones which can accommodate each one at least one of the requirements (containers) but maybe one of the zones has enough room for both requests, so here I want to left `paddingResources` at most of each NUMA zone so each zone can accommodate one request (container) but not both of them in the same zone.

baseLoad is calculated at node level ...so I do not know how should I handle it.
If we can assume that already running pods in the node where scheduled by TAS using numa-aware scheduling those resources should already be reflected into the node NRT, so there is no need to subtract any baseload, right?

Reply:
ok, concerning the base load. On each worker node we have a bunch of infra containers which are either best effort or burstable, and whose resources are not pinned on NUMA zones. And that's fine per se. The TAS accounting, however, is done considering guaranteed pods, because these are the only pods (actually containers in them, but still) which can get NUMA-pinned resources.

So we know the node has less resources then accounted, but we cannot account them properly to any NUMA zone, because of the awkward contractiction we have above.

There is not a right solution atm. The most conservative approach would be to deduct the base load from _all_ zones, but we don't do that. We usually account all these resources on a random NUMA zone.

Reply:
Ok that's what I thought ... Ok I will deduce it from one of the numa zones ... but if understand it properly, this still could make the test fail due to wrong padding if the zone we deduce the baseload from ends having enough room for the two containers.